src/misc_gui.cpp

```diff
@@ -913,7 +925,9 @@ struct QueryStringWindow : public Window
 		this->editbox.text.afilter = afilter;
 		this->flags = flags;
 
-		this->InitNested(WN_QUERY_STRING);
+		this->CreateNestedTree();
+		this->HideButtons();
```

Comment:
I don't think this needs to be a new function, since it's only called in one place.

Reply:
I don't understand this comment. InitNested() is an inherited function and can't be modified.

However, stuff to happen during the window init can just be put in the window's `OnInit()` function.